testinfra/test/test_modules.py

```diff
@@ -256,3 +256,11 @@ def test_ansible_module(TestinfraBackend, Ansible):
         assert variables["inventory_hostname"] == "debian_jessie"
         assert variables["ansible_user"] == "root"
         assert variables["group_names"] == ["ungrouped"]
+
+
+@all_images
+def test_mountpoint(MountPoint):
```

Comment:
just testing on debian_jessie should be enough here (just remove the `@all_images` decorator)